dbeam-core/src/main/java/com/spotify/dbeam/avro/JdbcAvroSchema.java

```diff
@@ -298,6 +316,34 @@ private static SchemaBuilder.FieldAssembler<Schema> createAvroFields(
     }
   }
 
+  private static SchemaBuilder.UnionAccumulator<SchemaBuilder.NullDefault<Schema>>
+      buildAvroFieldTypeFromPGType(
+      final String columnTypeName,
+      final boolean useLogicalTypes,
+      final SchemaBuilder.BaseTypeBuilder<
+          SchemaBuilder.UnionAccumulator<SchemaBuilder.NullDefault<Schema>>>
+          field) {
+    switch (columnTypeName) {
+      case "uuid":
+        if (useLogicalTypes) {
+          return field.stringBuilder().prop("logicalType", "uuid").endString();
+        } else {
```

Comment:
Does this handle nested array type? for instance array[array[int]]

Reply:
I think it won't handle that

Reply:
Good point, will add to docs